dirsrvtests/tests/suites/ds_logs/ds_logs_test.py

```diff
@@ -656,7 +656,7 @@ def test_internal_log_level_516(topology_st, add_user_log_level_516, disable_acc
                                     r'SRCH base="cn=group,ou=Groups,dc=example,dc=com".*')
     # (Internal) op=10(1)(2) ENTRY dn="cn=group,ou=Groups,dc=example,dc=com"
     assert topo.ds_access_log.match(r'.*\(Internal\) op=[0-9]+\([0-9]+\)\([0-9]+\) '
-                                    r'ENTRY dn="cn=group,ou=groups,dc=example,dc=com".*')
+                                    r'ENTRY dn="cn=group,ou=Groups,dc=example,dc=com".*')
```

Comment:
I would rather use match(..., flags=re.IGNORECASE)
because the test are sometime run against an older DS version

Reply:
This doesn't work since lib389 is overloading the match function.  It doesn't accept any other positional args.  So it would require a change to lib389 to get this to work, and that should be done in a separate ticket IMHO.